style.css

```diff
@@ -0,0 +1,832 @@
+  background: rgba(255, 255, 255, 0.1);
+  width: 90%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 10px;
+  padding: 1%;
+}
+
+.main-program-details-img {
+  width: 20%;
+}
+
+.main-program-details-heading {
+  color: #ec5242;
+  font-weight: 600;
+  font-size: 1.2rem;
+  width: 30%;
+}
+
+.main-program-details-description {
+  font-size: 0.8rem;
+  color: #fff;
+  line-height: 1.3;
+  width: 50%;
+  padding: 4px;
+}
+
+.main-program-button {
+  margin-top: 50px;
+  background-color: #ec5242;
+  border: none;
+  color: #fff;
+  width: 70%;
+  height: 60px;
+  font-size: 1.2rem;
+}
+
+.featuredSpeakersHeading {
+  color: #272a31;
+  font-weight: 600;
+  font-size: 1.5rem;
+  width: 100%;
+  text-align: center;
+  margin-top: 30px;
+}
+
+.featuredSpeakersBar1 {
+  margin-top: 10px;
+  width: 40px;
+  height: 2px;
+  background-color: #ec5242;
+  margin-left: 50%;
+}
+
+.featuredSpeakers {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 40px;
+}
+
+.featuredSpeakersContainer {
+  margin-top: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.featuredSpeakersImg img {
+  width: 178px;
+  height: 177px;
+}
+
+.featuredSpeakersBox {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.featuredSpeakersText {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-left: 15px;
+  justify-content: flex-start;
+}
+
+.featuredSpeakersName {
+  color: #272a31;
+  font-weight: 600;
+}
+
+.featuredSpeakersDetails1 {
+  color: #ec5242;
+  font-size: 0.8rem;
+  font-style: italic;
+  margin-top: 10px;
+}
+
+.featuredSpeakersBar {
+  margin-top: 8px;
+  width: 20px;
+  height: 2px;
+  background-color: #272a313d;
+}
+
+.featuredSpeakersDetails2 {
+  color: #272a31;
+  font-size: 0.8rem;
+  margin-top: 10px;
+}
+
+.partners,
+.about-partners {
+  background-color: #272a31;
+  margin-top: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.partners-heading {
+  margin-top: 20px;
+}
+
+.partners-heading h1 {
+  width: 100%;
+  text-align: center;
+  color: #ffffff9f;
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.partnersBar {
+  margin-top: 10px;
+  width: 30px;
+  height: 1px;
+  background-color: #ec5242;
+}
+
+.partnersLogo {
+  display: flex;
+  width: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-evenly;
+  margin-top: 30px;
+  gap: 30px;
+  margin-bottom: 30px;
+}
+
+.mozilla img,
+.google img,
+.airbnb img,
+.naver img,
+.twitter img {
+  width: 76px;
+  height: 40px;
+}
+
+.footer,
+.about-footer,
+.about-footer-768 {
+  width: 90%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 150px;
+  padding: 5%;
+}
+
+.footerLogo {
+  margin-top: 30px;
+  font-size: 0.8rem;
+}
+
+.footerDetails1 {
+  margin-top: 20px;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Please use a mobile-first approach. Use media queries to change the design from mobile to desktop, not the other way around. In general, that means that media queries use a min-width.

Reply:
Hello Francis,

Done.  Thanks